src/components/layout/Container.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+const Container = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <ContainerLayout>
+      <ContainerInnerBox>{children}</ContainerInnerBox>
+    </ContainerLayout>
+  );
+};
+
+const ContainerLayout = styled.div`
+  background-color: ${(props) => props.theme.colors.background};
+  width: ${(props) => props.theme.layout.maxWidth};
+  height: ${(props) => props.theme.layout.maxHeight};
+  transform: ${(props) => props.theme.layout.scale};
```

Comment:
```suggestion
  transform: scale(${(props) => props.theme.layout.scale});
```

이렇게 하면 비율도 해상도에 맞춰서 바뀝니다.

해보시고 괜찮으시면 침묵을.. 이상하다면 코멘트를 주세요

Reply:
![image](https://user-images.githubusercontent.com/76927397/212533642-56cc644f-5eb5-4be7-b0b1-2b5eace06c3b.png)

너무작아진다싶으면 저 1.83들을 1.84 1.85 이런식으로 조금씩 늘려가며 테스트해보시면 됩니다. 
혜연님 화면에 적절해보인다 싶으면 괜찮을 것 같습니다